client/src/store/toast.ts

```diff
@@ -0,0 +1,56 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { IToastProps } from "../components/UI/Toast";
+
+
+export enum Title{
+    SUCCESS = "Success",
+    ERROR = "Error",
+}
+
+declare interface IPayload{
+    type: string;
+    message:string;
+}
+
+let toastProperties:IToastProps;
+
+const initialToastState: {toastProp: IToastProps[] | []} = {
+    toastProp: []
+}
+
+const toastSlice = createSlice({
+    name: 'toast',
+    initialState:  initialToastState,
+    reducers:{
+        showToast(state,action: PayloadAction<IPayload>){
+            switch(action.payload.type) {
+                case 'Success':
```

Comment:
please make an enum for toast types and match against the enum,. hardcoding of string is never a good idea